federation/cmd/federation-apiserver/app/server.go

```diff
@@ -196,6 +197,8 @@ func Run(s *options.ServerRunOptions) error {
 		return err
 	}
 
+	routes.Version{}.Install(m.Mux, m.HandlerContainer)
```

Comment:
kubectl of course, yes.

I referred to this comment, haven't investigated where the values are overwritten with actual values in the build system: https://github.com/sttts/kubernetes/blob/5ead89b5bb1849d59589a972047e2876be479e62/pkg/version/base.go#L22-L22